__tests__/app.test.js

```diff
@@ -125,14 +129,73 @@ describe("/api/articles", () => {
             });
         });
-        test("Returns 'Status: 404' if invalid path provided to articles, e.g. misspelt path", () => {
-            return request(app).get('/api/articlees').expect(404);
+    });
+});
+
+describe("/api/articles/:article_id", () => {
+    describe("GET", () => {
+        test("Returns 'Status: 200' with single article object if valid article ID", () => {
+            return request(app).get('/api/articles/1').expect(200)
+            .then(({ body }) => {
+                const article = body.requestedArticle;
+                expect(typeof article).toBe("object");
+                expect(Array.isArray(article)).toBe(false);
+            });
+        });
+        test("Returns an article object containing the correct keys", () => {
+            return request(app).get('/api/articles/1').expect(200)
+            .then(({ body }) => {
+                const article = body.requestedArticle;
+                expect(article).toHaveProperty("author", expect.any(String));
+                expect(article).toHaveProperty("title", expect.any(String));
+                expect(article).toHaveProperty("article_id", expect.any(Number));
+                expect(article).toHaveProperty("body", expect.any(String));
+                expect(article).toHaveProperty("topic", expect.any(String));
+                expect(article).toHaveProperty("created_at", expect.any(String));
+                expect(article).toHaveProperty("votes", expect.any(Number));
+                expect(article).toHaveProperty("article_img_url", expect.any(String));
+            });
+        });
+        test("Returns the correct article object for article ID 1", () => {
+            return request(app).get('/api/articles/1').expect(200)
+            .then(({ body }) => {
+                const article = body.requestedArticle;
+                expect(article).toHaveProperty("author", "butter_bridge");
+                expect(article).toHaveProperty("title", "Living in the shadow of a great man");
+                expect(article).toHaveProperty("article_id", 1);
+                expect(article).toHaveProperty("body", "I find this existence challenging");
+                expect(article).toHaveProperty("topic", "mitch");
+                expect(article).toHaveProperty("created_at", "2020-07-09T20:11:00.000Z");
+                expect(article).toHaveProperty("votes", 100);
+                expect(article).toHaveProperty("article_img_url", "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700");
+            });
         });
-        test("Returns custom error message if invalid path provided ", () => {
-            return request(app).get('/api/articlees').expect(404).then((body) => {
-                const errorMsg = body.error.text
-                expect(errorMsg).toBe("Invalid path provided - please try again");
+        test("Returns the correct article object for other valid article IDs", () => {
```

Comment:
There's no need to do multiple tests for this - as long as it works for one, you've proven the concept!